pizza_box/tests/test_handlers.py

```diff
@@ -1,2 +1,42 @@
-def test_imports():
-    from pizza_box.handlers import APBBinFileHandler  # noqa F401
+import pkg_resources
+import pytest
+import pandas
+
+from pizza_box.handlers import APBBinFileHandler
+
+
+basename = "ff2df6b8-eff5-4bb5-9718-69698ab6fa3c"
+
+
+def _get_file(basename, extension):
+    file = pkg_resources.resource_filename("pizza_box", f"tests/example_data/{basename}.{extension}")
+    return file
+
+
+@pytest.fixture
+def bin_file():
+    bin_file = _get_file(basename, "bin")
+    return bin_file
+
+
+@pytest.fixture
+def txt_file():
+    txt_file = _get_file(basename, "txt")
+    return txt_file
+
+
+def test_handler_init(bin_file, txt_file):
```

Comment:
Is `txt_file` going to be tested in this function?

Reply:
Good point. I think it's not needed right now, as it's implicitly used inside the class (not an input yet, but probably we should change it).

Reply:
I removed them.